### ISSUE TITLE
Improve text color discoverability by using the industry name "Text color" and the "A" icon

### ISSUE THREAD

**Issue by paulopmt1** [WordPress/gutenberg#64539]
## What problem does this address?

Currently, our text color button is called "Highlight" which doesn't reflect its function properly:

<img width="482" alt="image" src="https://github.com/user-attachments/assets/70f27cc0-196a-4f3a-bbd1-e3515c7beb6c">

## What is your proposed solution?

Historically, software editors use the `A` icon to change a text color. My proposal is that we change:

- The icon using the default `A` one
- The name for `Text color` instead of `Highlight`

### Some examples 

- Canva uses "Text color" with a colorful icon:

<img width="487" alt="image" src="https://github.com/user-attachments/assets/1d11aba9-fd07-460e-bfdf-0547b89b5e61">

- Google docs also uses "Text color":

<img width="265" alt="image" src="https://github.com/user-attachments/assets/903266be-6ebf-417f-8936-9666514064a0">

**Comment by t-hamano** [comment]
Thanks for the report.

The Highlight allows us to change both the text color and the text background color. If they are changed, the icon will change to an "A" icon and the text color and background color will be applied:

![image](https://github.com/user-attachments/assets/c4d87502-abc2-4825-a3b5-e92e0b15fc02)

I would be grateful if you could let me know if there is anything we can improve about this UI.

**Comment by paulopmt1** [comment]
Oh, now I understand why we call it Highlight. In that case, I'd keep the Highlight and add a new Item called `Text color` before it since it's more used than the Highlight itself.

If adding another menu is a problem, I'd just rename it to `Text color` (and change the icon) because is better to have a `highlight` feature hidden inside a `Text color` than the opposite as we have currently, as changing the text color is something more used than changing a text background color.

For future readers, this is what @t-hamano is talking about, we have two tabs inside the same button: 

<img width="339" alt="image" src="https://github.com/user-attachments/assets/588edc04-2426-41b8-a249-e71813fa07ff">

**Comment by snehapatil2001** [comment]
@paulopmt1 @t-hamano I propose renaming the 'Highlight' button to 'Color'. This would make the interface clearer and more intuitive by grouping both color-related options under a single, neutral label.

**Comment by t-hamano** [comment]
@snehapatil2001 Thank you for your suggestion.

In #34680, where this feature was implemented, the original label was "Color", but it seems that the final label was decided to be "Highlight" based on the feedback from [this comment](https://github.com/WordPress/gutenberg/pull/34680#discussion_r714576936).

I think we need copy feedback and design feedback on whether to change this label to "Color". Personally, I think the label "color" is easier to understand.

cc @WordPress/gutenberg-design @annezazu

**Comment by jasmussen** [comment]
I'd not rename to _Color_, as that's used for the panel and refers to colors of a block level nature. "Text color" or "Highlight color" could be fine. 

An icon change can also work, as can the "A", though I would not change the colors of the icon itself as [here](https://github.com/WordPress/gutenberg/issues/64539#issuecomment-2291461278), it doesn't have precedence in the UI elsewhere to do that, and might disorient. What we can do is the same as the examples mentioned, where there's a color-bar (or two, if you want to show both foreground and background) below the A.

**Comment by t-hamano** [comment]
This is just a simple mockup, but does it make sense to have separate text and highlight colors?

![image](https://github.com/user-attachments/assets/9c44e423-7623-4a19-962e-df91b8b6070b)

I used Google Docs as a reference:

![image](https://github.com/user-attachments/assets/d21e0434-c0b1-4ee6-aa57-b086295cf4b7)

**Comment by jasmussen** [comment]
Might work.

**Comment by paulopmt1** [comment]
> does it make sense to have separate text and highlight colors?

This seems ideal, so users don't need to click on one of them to learn there is a "Text" or "Background" color to change.

**Comment by autumnfjeld** [comment]
> This is just a simple mockup, but does it make sense to have separate text and highlight colors?

Yes, this mockup is great @t-hamano. It is so much clearer.  This labeling follows what is standard in editing software.

**Comment by kristastevens** [comment]
Hello from Editorial! 👋  Just jumping in to respond to the "Needs Copy Review" ping. 

It looks like you've come up with a great solution here—no input needed on our end.

**Comment by davemart-in** [comment]
Jumping in to share that 3 out of 5 folks that I asked to change the color of text in a recent usability test struggle to locate where to do it. My hypothesis is that "Highlight" is not descriptive enough.

I'd propose using "Highlight color".

**Comment by jasmussen** [comment]
There seems to be general agreement, and next step could be a simple PR. I'd be happy to review, if that helps.

**Comment by dhanson-wp** [comment]
I agree with @t-hamano [above](https://github.com/WordPress/gutenberg/issues/64539#issuecomment-2297940958). I would recommend the most common UX writing for this is "Text color" instead of "Highlight color." Having both options visible in the dropdown options (Text and Highlight) would be ideal. This will be a great update.

**Comment by tyxla** [comment]
I like the simplicity of having immediate two separate options for text and background color. Happy to help with reviewing a PR.

**Comment by t-hamano** [comment]
There is an issue that needs to be resolved before this issue can move forward.

The current dropdown is sorted by title. This is not ideal, as it means the sorting will change depending on the locale. If we split the color control into two, it will be difficult to place them next to each other.

<img width="186" height="383" alt="Image" src="https://github.com/user-attachments/assets/e0471faf-6051-429a-81cb-37369ef4a350" />

We probably want a prop to control the rendering order of `RichTextToolbarButton`s.

**Comment by tyxla** [comment]
How big a blocker is the sort order, really, considering that it's not something new, and it's already the case when you change to a different locale? I'd suggest proceeding regardless.
